object/check.go

```diff
@@ -34,6 +34,8 @@ const (
 
 	DefaultFailedSigninLimit      = 5
 	DefaultFailedSigninFrozenTime = 15
+
+	disabledLdapUserErrorData = "data 533"
```

Comment:
confused about how we check disabled ldap user,  @yamaritta Did you this approach before?

Reply:
the problem is that if the user, for example, does not exist and if the user is blocked, then in both cases we will get errors that will differ by the `data` field in the error line

Reply:
yes, we blocked user in  database when sync from ldap

Reply:
but in our case ldap library helped us. if the current implementation does not allow get  account status, can leave the check as is

Reply:
now lgfm, @yamaritta what about you?

Reply:
I remember userAccountControl is a bitmask. I'm not sure that this comparison is correct
https://learn.microsoft.com/ru-ru/troubleshoot/windows-server/active-directory/useraccountcontrol-manipulate-account-properties

Reply:
For a regular blocked account, the value of userAccountControl will always be 514:

`NORMAL_ACCOUNT (512) + ACCOUNTDISABLE (2) = 514`

Reply:
Ok